change-control-lambda/test/time-window.test.ts

```diff
@@ -58,3 +67,28 @@ test('a blocked window starts AND finishes within margin', () => {
   const x = shouldBlockPipeline(ics, new Date('2019-02-03T07:00:00.000Z'), 72 * 3_600);
   expect(x && x.summary).toBe('Block3');
 });
+
+test('left edge for recurring event after the initial event', () => {
```

Comment:
I'm not quite following these test descriptor phrases, and what test cases are being covered.

Can you better phrase, or add any additional comments? What's the initial event and the recurring event here? 
Phrases such as 'current time aligns with the left edge of the first occurrence' might be clearer to understand.

Reply:
Cool, makes sense. I'll improve the descriptions/add comments.